guava/src/com/google/common/util/concurrent/Monitor.java

```diff
@@ -1031,7 +1024,7 @@ private boolean isSatisfied(Guard guard) {
       return guard.isSatisfied();
     } catch (Throwable throwable) {
       signalAllWaiters();
-      throw Throwables.propagate(throwable);
+      throw throwable;
```

Comment:
I had apparently missed/forgotten that this was possible [since Java 7](https://docs.oracle.com/javase/7/docs/technotes/guides/language/catch-multiple.html) -- nice.

Reply:
Yeah. I'm sensitive to Throwables.propagate() because we actually had an exception-handling bug in Monitor's early days due to overuse of it, which was fixed with a more involved set of changes in 287bc67cac97052b13cbbc0358aed8054b14bd4a (long before Java 8 came out).